big_tests/tests/connect_SUITE.erl

```diff
@@ -79,30 +80,33 @@ groups() ->
     ct_helper:repeat_all_until_all_ok(G).
 
 tls_groups()->
-    [{group, c2s_noproc},
-     {group, starttls},
+    [{group, starttls},
+     {group, c2s_noproc},
```

Comment:
Meaningless change. I was reorganising groups and these two ended up in different order but it doesn't change anything.